gnomad/assessment/sanity_checks.py

```diff
@@ -0,0 +1,166 @@
+import logging
+from typing import Dict, List, Optional
+
+import hail as hl
+
+from gnomad.utils.vcf import make_label_combos, SORT_ORDER
+
+
+logging.basicConfig(format="%(levelname)s (%(name)s %(lineno)s): %(message)s")
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+def generic_field_check(
+    ht: hl.Table,
+    cond_expr: hl.expr.BooleanExpression,
+    check_description: str,
+    display_fields: List[str],
+    verbose: bool,
+) -> None:
+    """
+    Check a generic logical condition involving annotations in a Hail Table and print the results to terminal.
+
+    Displays the number of rows in the Table that match the `cond_expr` and fail to be the desired condition (`check_description`). 
+    If the number of rows that match the `cond_expr` is 0, then the Table passes that check; otherwise, it fails.
+
+    .. note::
```

Comment:
Thank you for adding this note -- it's probably something easily missed when setting up the generic field check, so it's great to have this additional reminder